enhancements/monitoring/monitoring-config.md

```diff
@@ -0,0 +1,255 @@
+---
+title: monitoring-config 
```

Comment:
this has an additional trailing space after monitoring-config

Reply:
done